addons/account/models/chart_template.py

```diff
@@ -243,7 +243,7 @@ def _pre_reload_data(self, company, template_data, data):
                     journal = self.env['account.journal'].with_context(active_test=False).search([
                         ('code', 'in', (journal_data['code'], translated_code)),
                         ('company_id', '=', company.id),
-                    ])
+                    ], limit=1)
```

Comment:
Had  bit of a talk with @svfu-odoo and @Levizar to all agree that while this whole function is a bit mysterious, your solution makes sense :)
However, it seems to turn out that the `_update_xmlids` (line 257)  does not seem to be helpful, nor needed, in the context of a chart template reupload. We can probably just get rid of it, and this task seems to be the opportunity to do so.  Could you please have a look ?

Reply:
The point of this is to avoid recreating the journal if it was created manually.
Otherwise we would have a unique constraint error because 2 journals have the same code.